index.bs

```diff
@@ -2639,61 +2639,65 @@ When registering a new credential, represented by a {{AuthenticatorAttestationRe
 
-9. Determine the attestation statement format by performing an USASCII case-sensitive match on |fmt| against the set of
+1. Determine the attestation statement format by performing an USASCII case-sensitive match on |fmt| against the set of
     supported WebAuthn Attestation Statement Format Identifier values.
     The up-to-date list of registered WebAuthn Attestation Statement Format Identifier values
     is maintained in the in the IANA registry of the same name [[!WebAuthn-Registries]].
 
-10. Verify that |attStmt| is a correct [=attestation statement=], conveying a valid [=attestation signature=], by using the
+1. Verify that |attStmt| is a correct [=attestation statement=], conveying a valid [=attestation signature=], by using the
     [=attestation statement format=] |fmt|'s verification procedure given |attStmt|, |authData| and the [=hash of the serialized
     client data=] computed in step 6.
 
     Note: Each [=attestation statement format=] specifies its own verification procedure. See [[#defined-attestation-formats]] for
     the initially-defined formats, and [[!WebAuthn-Registries]] for the up-to-date list.
 
-11. If validation is successful, obtain a list of acceptable trust anchors (attestation root certificates or [=ECDAA-Issuer
+1. If validation is successful, obtain a list of acceptable trust anchors (attestation root certificates or [=ECDAA-Issuer
     public key=]s) for that attestation type and attestation statement format |fmt|, from a trusted source or from policy. For
     example, the FIDO Metadata Service [[FIDOMetadataService]] provides one way to obtain such information, using the
     <code>[=aaguid=]</code> in the <code>[=attestedCredentialData=]</code> in |authData|.
 
-12. Assess the attestation trustworthiness using the outputs of the verification procedure in step 10, as follows:
+1. Assess the attestation trustworthiness using the outputs of the verification procedure in step 10, as follows:
         - If [=self attestation=] was used, check if [=self attestation=] is acceptable under [=[RP]=] policy.
         - If [=ECDAA=] was used, verify that the [=identifier of the ECDAA-Issuer public key=] used is included in the set of
             acceptable trust anchors obtained in step 11.
         - Otherwise, use the X.509 certificates returned by the verification procedure to verify that the attestation public key
             correctly chains up to an acceptable root certificate.
 
-13. If the attestation statement |attStmt| verified successfully and is found to be trustworthy, then register the new
+1. Check that the <code>[=credentialId=]</code> is not yet registered to any other user. If registration 
+    is requested for a credential that is already registered to a different user, the [=[RP]=] SHOULD 
+    fail this ceremony, or it MAY decide to accept the registration, e.g. while deleting the older registration.
```

Comment:
s/this ceremony /this [=registration=] ceremony /

Reply:
Done.